client/client-multi/client-cpp/CMakeLists.txt

```diff
@@ -204,6 +204,10 @@ if(NOT MSVC)
    set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -ggdb")
 endif()
 
+# "no-unused-private-field" is added since in most of cases it is false positive
+list(APPEND KAA_COMPILE_OPTIONS
+                -Wno-unused-private-field)
```

Comment:
Well, I had removed any mention of IKaaClientContext from ProfileManager.hpp and had edited KaaClient.cpp and SDK successfully compiled. Did I understand you correctly?